CONTRIBUTING.md

```diff
@@ -1,102 +1,181 @@
+To start contributing to the Sanity Studio, you need to set it up locally first. Fork this repository and clone it onto to your computer in your desired location.
 
-## Publishing _prerelease_ versions from `next`
+```
+git clone git@github.com:sanity-io/sanity.git
+```
 
-This is done by running `npm run publish` from the project root, selecting "Prerelease" from the menu and choosing a _Prerelease identifier_ (e.g. **alpha**, **beta** or **rc**). A prerelease can be tested in the studio by running e.g. `sanity upgrade --tag=beta --save-exact`
+Then, at the root of the Sanity project folder, install dependencies with `npm install` from the terminal. When all dependencies have been installed, run `npm run init` to initialize the project.
+
+```
+npm install
+npm run init
+```
 
-## Publishing _canary_ versions from feature branches
+To start the Sanity Studio you need to configure the Test Studio (packages/test-studio) first. Add your own project ID and dataset name to the `sanity.json` config of the Test Studio (`packages/test-studio/sanity.json`) and save your changes.
 
-This can be done at any time by anyone and is done by `npm run publish-canary`. This will publish with the commit hash from HEAD, and can be installed with `sanity upgrade --tag=canary --save-exact`
+Note that the Test Studio uses [spaces](https://www.sanity.io/docs/experimental/spaces), which is currently an experimental feature. If you want to test the Studio without this, remove the entire `__experimental_spaces` array. If you would like to use the Test Studio with spaces, be sure to create [datasets](https://www.sanity.io/docs/data-store/datasets) to use for each space and provide the appropriate names and titles. Each space should have a dataset name, and optionally a project ID.
 
-# How to file an issue
+```
+// sanity.json
+"__experimental_spaces": [
+  {
+    "name": "yourDatasetName1",
+    "title": "Your Dataset Name 1",
+    "default": true,
+    "api": {
+    	"dataset": "yourDataset1"
+  	}
+  },
+  {
+    "name": "yourDatasetName2",
+    "title": "Your Dataset Name 2",
+    "api": {
+      "dataset": "yourDataset2"
+    }
+  }
+],
+"api": {
+  "projectId": "yourProjectId",
+  "dataset": "yourDataset1"
+}
+```
 
-If you find a security vulnerability, do **NOT** open an issue. Email security@sanity.io instead.
+Next, start the Test Studio by running `npm run start` in the terminal (while still in the root folder) and open it in your favorite browser to see it live.
 
-Any security issues should be submitted directly to security@sanity.io. In order to determine whether you are dealing with a security issue, ask yourself these two questions:
+```
+npm run start
+```
 
-- Can I access something that's not mine, or something I shouldn't have access to?
-- Can I disable something for other people?
+## Working on fixes or features
```

Comment:
Would it be wise to mention what kind of naming we want on the branch? We could for example say that they must start with `contribution/<branch-name-here>` or something similar.